Cargo.lock

```diff
@@ -407,50 +526,66 @@ checksum = "baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd"
 
 [[package]]
 name = "clap"
-version = "4.4.18"
+version = "4.5.3"
```

Comment:
@koxu1996

Reply:
It was already updated to the same version [here](https://github.com/cspr-rad/kairos/pull/36/files#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87eL410), and there was no warning in CI/CD (there is no warning when building locally as well).

Reply:
Okay, I see no clap update now :relieved:.